EIPS/eip-1.md

```diff
@@ -183,7 +183,9 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 #### `requires` header
 
-EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
+EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on. If such a dependency exists, this field is required.
+
+A `requires` dependency is created when the current EIP cannot be understood or implemented without a concept or technical element from another EIP. Merely mentioning another EIP does not necessarily create such a dependency.
```

Comment:
Good. Happy that my poor English helps.

Reply:
I support this principle, so long as we also require authors keep Specification as minimum as possible: Leave anything that's rationale, motivation or reference implementations _out of_ Specification section.

Reply:
EIP-1 doesn't really use RFC 2119 (and related), so I don't really like using the keywords in all caps.

Reply:
Just curious, what was our rationale that EIP-1 didn't want to adopt RFC 2119? 
I don't have strong opinion to preserve the cap, but I weakly prefer having a cap make it more obvious it is a rule rather than just a description.

Reply:
> Just curious, what was our rationale that EIP-1 didn't want to adopt RFC 2119?

Readability, most likely. I definitely would support adopting RFC 2119 for EIP-1 though.

Reply:
> Probably out of scope for this PR though.

~~probably~~ definitely